packages/app-blueprint/files/package.json

```diff
@@ -60,7 +60,7 @@
     "broccoli-asset-rev": "^3.0.0",
     "concurrently": "^9.1.2",
     "ember-auto-import": "^2.10.0",
-    "ember-cli": "~<%= emberCLIVersion %>",
+    "ember-cli": "~6.4.0-beta.1",
```

Comment:
So this would need to be bumped along with other packages?

Reply:
yes exactly, I would need to add that to the script thanks for the reminder

Reply:
in fact this will automatically be bumped by release-plan, I have configured it correctly now to match the main ember-cli version 👍